README.rst

```diff
@@ -102,7 +102,7 @@ Partial example: generating a C struct
         *
         * Auto-generated, do not edit.
         *
-        * Source file: {{T.source_file_path}}
+        * Source file: {{T.source_file_path | replace("\\", "\\\\")}}
```

Comment:
We wouldn't need to do this if source_file_path was a `pathlib.Path`, right? Perhaps that's the better fix; to convert this into pathlib?

Reply:
This is a quick fix, the issue has been documented for over a year. There is plenty of time to fix it properly afterwards, i may even give it a shot.

Reply:
>We wouldn't need to do this if source_file_path was a pathlib.Path, right? Perhaps that's the better fix; to convert this into pathlib?

Check this out: https://github.com/UAVCAN/pydsdl/pull/77

We would have to use `.as_posix()` though to ensure consistent behavior.